Developers/Unity Project/Spooksville/Assets/Scripts/Items/Weapon.cs

```diff
@@ -2,16 +2,33 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public abstract class Weapon
+public class Weapon
```

Comment:
If you add a `[System.Serializable]` annotation at the top here, i believe this will become visible in the editor and saved to the scene/prefab. See other comment for integration tip.